[2024-05-04T11:43:07.903+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T11:43:08.133+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.gcs_to_bq_geolocation scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:43:08.188+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.gcs_to_bq_geolocation scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:43:08.189+0000] {taskinstance.py:2303} INFO - Starting attempt 6 of 6
[2024-05-04T11:43:08.257+0000] {taskinstance.py:2327} INFO - Executing <Task(GCSToBigQueryOperator): gcs_to_bq_geolocation> on 2024-05-03 00:00:00+00:00
[2024-05-04T11:43:08.462+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=13567) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-04T11:43:08.463+0000] {standard_task_runner.py:63} INFO - Started process 13713 to run task
[2024-05-04T11:43:08.425+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'gcs_to_bq_geolocation', 'scheduled__2024-05-03T00:00:00+00:00', '--job-id', '247', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpeqoxgc0n']
[2024-05-04T11:43:08.474+0000] {standard_task_runner.py:91} INFO - Job 247: Subtask gcs_to_bq_geolocation
[2024-05-04T11:43:09.085+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.gcs_to_bq_geolocation scheduled__2024-05-03T00:00:00+00:00 [running]> on host d4c1d9b975b8
[2024-05-04T11:43:13.738+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='gcs_to_bq_geolocation' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-03T00:00:00+00:00'
[2024-05-04T11:43:13.739+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-04T11:43:14.050+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-04T11:43:14.249+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-04T11:43:14.327+0000] {gcs_to_bigquery.py:379} INFO - Using existing BigQuery table for storing data...
[2024-05-04T11:43:15.555+0000] {bigquery.py:2328} INFO - Project is not included in destination_project_dataset_table: order.olist_geolocation_dataset; using project "project-capstone-422211"
[2024-05-04T11:43:15.555+0000] {gcs_to_bigquery.py:383} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'project-capstone-422211', 'datasetId': 'order', 'tableId': 'olist_geolocation_dataset'}, 'sourceFormat': 'CSV', 'sourceUris': ['gs://storage-capstone/olist_geolocation_dataset.csv'], 'writeDisposition': 'WRITE_TRUNCATE', 'ignoreUnknownValues': False, 'skipLeadingRows': None, 'fieldDelimiter': ',', 'quote': None, 'allowQuotedNewlines': False, 'encoding': 'UTF-8'}}
[2024-05-04T11:43:15.580+0000] {bigquery.py:1613} INFO - Inserting job ***_etl_gcs_to_bq_geolocation_2024_05_03T00_00_00_00_00_fee9fbc978b5cfc9e48a4bc04aecf788
[2024-05-04T11:43:49.603+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-04T11:43:49.610+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=gcs_to_bq_geolocation, execution_date=20240503T000000, start_date=20240504T114308, end_date=20240504T114349
[2024-05-04T11:43:49.648+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-04T11:43:49.664+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-04T11:43:49.671+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
